db/1-schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS ACCOUNTS (
```

Comment:
Definitely needs to be considered. I think the goal was to get something out there this first cycle :)